File: VB/NetCore/WhatIsMyBrowser.CommonTypesCoreVB/UserAgentMetaData.vb
Imports Newtonsoft.Json

Public Class UserAgentMetaData
    <JsonProperty("id")>
    Public Property Id As Long

    <JsonProperty("times_seen")>
    Public Property TimesSeen As Long

    <JsonProperty("first_seen_at")>
    Public Property FirstSeenAt As DateTimeOffset

    <JsonProperty("last_seen_at")>
    Public Property LastSeenAt As DateTimeOffset
End Class

File: CS/NET40/WhatIsMyBrowser.CommonTypes/UserAgentMetaData.cs
using System;
using Newtonsoft.Json;

namespace WhatIsMyBrowser.CommonTypes
{
    public class UserAgentMetaData
    {
        [JsonProperty("id")]
        public long Id { get; set; }

        [JsonProperty("times_seen")]
        public long TimesSeen { get; set; }

        [JsonProperty("first_seen_at")]
        public DateTimeOffset FirstSeenAt { get; set; }

        [JsonProperty("last_seen_at")]
        public DateTimeOffset LastSeenAt { get; set; }
    }
}